spec/controllers/admin/daily_questions_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe Admin::DailyQuestionsController do
+  login_admin
```

Comment:
See the comments bellow for this method, and wrap in inside a `before` block, in the test suite.